src/04-arrays.js

```diff
@@ -65,7 +74,13 @@ function getFirstElementOfArray(array) {}
  *  getLastElementOfArray([ null, undefined ]);
  *  //> undefined
  */
-function getLastElementOfArray(array) {}
+function getLastElementOfArray(array) {
+
+  return array.pop();
+}
```

Comment:
This is problematic in that it actually removes the last element of the array, which would be an unexpected result when you're asked to "Return" the element, not "Remove". Could introduce some serious bugs!

I would try to rewrite this one without `pop`. How can you access the last index of the array? The JSDoc has a hint!